### ISSUE TITLE
Remove jQuery

### ISSUE THREAD

**Issue by davidism** [pallets/werkzeug#1807]
At this point, support for modern JavaScript features in modern browsers is probably enough to replace jQuery with plain JavaScript. Bundling jQuery, even minimized, is 1/4 of the final size of the wheel on PyPI. We also keep getting asked to update jQuery due to CVEs that come out, even though they have no impact on the debugger.

**Comment by davidism** [comment]
We should probably support the same browser versions as jQuery: https://jquery.com/browser-support/. If IE is too much of a problem, I'm probably willing to drop it, it only accounts for 0.3% of page views (for docs at least).

**Comment by ThiefMaster** [comment]
I'd just drop everything that's not modern. It's a developer tool, why bother with old crap?

Modern Firefox, Chrome, Edge, Safari and that's it.

**Comment by davidism** [comment]
We need some sort of baseline, and that's probably fine too (it basically matches jQuery except for IE). We probably should account for Firefox ESR at least.

**Comment by ThiefMaster** [comment]
I doubt ESR is relevant for developer tool. Anyway the next ESR major release is coming out next month and I don't think anything important changed in the last year that's relevant for the debugger.

**Comment by davidism** [comment]
The only feature I was thinking of off the top of my head was fetch vs XHR, since fetch is much nicer to work with but excludes IE. Yeah, let's go with completely modern for now.

**Comment by bclodius** [comment]
> We also keep getting asked to update jQuery due to CVEs that come out, even though they have no impact on the debugger.

This is a huge painpoint on my end.  Every few weeks / months all my python projects explode with new security finding due to this jquery dependency.

```
find venv -name jquery*
venv/lib/python3.7/site-packages/werkzeug/debug/shared/jquery.js
```

**Comment by juped** [comment]
@kx-chen and @chrisngyn will be working on this!

**Comment by obfusk** [comment]
Seems to be fixed by #1857.

**Comment by Arjunsankarlal** [comment]
@davidism @ThiefMaster I saw in an issue that a separate release won't be made due to jQuery version updation alone and it made sense. But now it is been deleted as it is not necessary and lot of them are facing this security issue because of the jQuery version in the current stable release, though mostly debugging in live won't happen or only trusted people would do. It would be nice if a patch release is made for this. This would help a lot of devs to avoid the security issues raised by the scanners and security teams.

TIA.
